src/crimson/os/cyan_store.cc

```diff
@@ -4,6 +4,228 @@
 
 #include "common/safe_io.h"
 
+#include "crimson/os/cyan_collection.h"
+#include "crimson/os/cyan_object.h"
+#include "crimson/os/Transaction.h"
+
+namespace {
+  seastar::logger& logger() {
+    return ceph::get_logger(ceph_subsys_filestore);
+  }
+}
+
+namespace ceph::os {
+
+using ObjectRef = boost::intrusive_ptr<Object>;
+
+CyanStore::CyanStore(const std::string& path)
+  : path{path}
+{}
+
+CyanStore::~CyanStore() = default;
+
+seastar::future<> CyanStore::mount()
+{
+  bufferlist bl;
+  string fn = path + "/collections";
+  string err;
+  if (int r = bl.read_file(fn.c_str(), &err); r < 0) {
+    throw std::runtime_error("read_file");
+  }
+
+  set<coll_t> collections;
+  auto p = bl.cbegin();
+  decode(collections, p);
```

Comment:
Hmm, have the format diverged  from `MemStore`?

Reply:
no, i don't think so.